test/e2e/tickets/fetch_matches_test.go

```diff
@@ -16,21 +16,29 @@
 package tickets
 
 import (
-	"fmt"
 	"testing"
+	"time"
 
 	structpb "github.com/golang/protobuf/ptypes/struct"
 	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	statestoreTesting "open-match.dev/open-match/internal/statestore/testing"
 	"open-match.dev/open-match/internal/testing/e2e"
 	"open-match.dev/open-match/pkg/pb"
 )
 
 func TestGameMatchWorkFlow(t *testing.T) {
+	var gotFmResp *pb.FetchMatchesResponse
```

Comment:
need to declare all of these upfront? check if avoidable - if not, may be declare close to first use? Havent see golang code have all variables declared uptop a function.

Reply:
Sometimes I have to declare these variables instead of using inline declaration because of golangci's shadow check, yet it looks cumbersome both having the `var` declarations with inline ones in one function so I just move everything upfront and try to use `var` instead. I'll try to figure out an alternative.